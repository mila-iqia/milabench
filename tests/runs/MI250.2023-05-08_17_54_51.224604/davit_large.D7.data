{"event": "config", "data": {"dirs": {"base": "/milabench/envs", "venv": "/milabench/envs/venv/torch", "data": "/milabench/envs/data", "runs": "/milabench/envs/runs", "extra": "/milabench/envs/extra/timm", "cache": "/milabench/envs/cache"}, "arch": "rocm", "group": "timm", "install_group": "torch", "install_variant": "rocm", "run_name": "redizilo.2023-05-08_17:54:51.224604", "enabled": true, "capabilities": {"nodes": 1}, "max_duration": 600, "voir": {"options": {"stop": 60, "interval": "1s"}}, "config_base": "/milabench/milabench/config", "config_file": "/milabench/milabench/config/standard.yaml", "definition": "/milabench/milabench/benchmarks/timm", "plan": {"method": "per_gpu"}, "argv": {"--amp": true, "--model": "davit_large", "--batch-size": 128, "--lr-base": 0.01}, "tags": ["classification", "transformer", "vision"], "weight": 1.0, "name": "davit_large", "tag": ["davit_large", "D7"], "device": 7, "devices": [7], "env": {"ROCR_VISIBLE_DEVICES": "7"}}, "pipe": null}
{"event": "start", "data": {"command": ["voir", "--config", "/milabench/envs/extra/timm/voirconf-davit_large.D7-0efae956f1553a76c1e03985181900f5.json", "/milabench/milabench/benchmarks/timm/pytorch-image-models/train.py", "--amp", "--model", "davit_large", "--batch-size", "128", "--lr-base", "0.01", "--data-dir", "/milabench/envs/data", "--dataset", "FakeImageNet", "--output", "/milabench/envs/extra/timm/redizilo.2023-05-08_17:54:51.224604/davit_large.D7", "--checkpoint-hist", "1"], "time": 1683570775.3605828}, "pipe": null}
{"event": "phase", "data": {"name": "init"}, "pipe": "data"}
{"event": "phase", "data": {"name": "parse_args"}, "pipe": "data"}
{"event": "phase", "data": {"name": "load_script"}, "pipe": "data"}
{"event": "phase", "data": {"name": "run_script"}, "pipe": "data"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "Training with a single process on 1 device (cuda:0).\n", "pipe": "stderr"}
{"event": "line", "data": "Model davit_large created, param count:196811752\n", "pipe": "stderr"}
{"event": "line", "data": "Data processing configuration for current model + dataset:\n", "pipe": "stderr"}
{"event": "line", "data": "\tinput_size: (3, 224, 224)\n", "pipe": "stderr"}
{"event": "line", "data": "\tinterpolation: bicubic\n", "pipe": "stderr"}
{"event": "line", "data": "\tmean: (0.485, 0.456, 0.406)\n", "pipe": "stderr"}
{"event": "line", "data": "\tstd: (0.229, 0.224, 0.225)\n", "pipe": "stderr"}
{"event": "line", "data": "\tcrop_pct: 0.95\n", "pipe": "stderr"}
{"event": "line", "data": "\tcrop_mode: center\n", "pipe": "stderr"}
{"event": "line", "data": "Learning rate (0.005) calculated from base learning rate (0.01) and global batch size (128) with linear scaling.\n", "pipe": "stderr"}
{"event": "line", "data": "Using native Torch AMP. Training in mixed precision.\n", "pipe": "stderr"}
{"event": "line", "data": "Scheduled epochs: 300. LR stepped per epoch.\n", "pipe": "stderr"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [839, 65520], "load": 0.0, "temperature": 41.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.224273681640625}, "pipe": "data"}
{"event": "line", "data": "Train: 0 [   0/32 (  0%)]  Loss: 7.224 (7.22)  Time: 5.712s,   22.41/s  (5.712s,   22.41/s)  LR: 1.000e-05  Data: 0.755 (0.755)\n", "pipe": "stderr"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [22954, 65520], "load": 0.4, "temperature": 44.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.176422595977783}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32306, 65520], "load": 0.33, "temperature": 46.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.276222229003906}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.335570335388184}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32306, 65520], "load": 1.0, "temperature": 47.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.255920886993408}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32306, 65520], "load": 1.0, "temperature": 48.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.161981582641602}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32306, 65520], "load": 1.0, "temperature": 49.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.255996227264404}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.4219651789743, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [1, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32308, 65520], "load": 0.35, "temperature": 48.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.163295745849609}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.444408930664515, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [2, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.3082275390625}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32308, 65520], "load": 1.0, "temperature": 49.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.512827118653625, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [3, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.283428192138672}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32308, 65520], "load": 1.0, "temperature": 50.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.523799731179544, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [4, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.234785556793213}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32308, 65520], "load": 1.0, "temperature": 51.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.426561627897634, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [5, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.196159362792969}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.44024959056773, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [6, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32308, 65520], "load": 0.99, "temperature": 50.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.325262069702148}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.589203999230996, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [7, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.243462085723877}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32308, 65520], "load": 1.0, "temperature": 51.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.41718059813868, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [8, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.18347692489624}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32308, 65520], "load": 1.0, "temperature": 51.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.30789902212263, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [9, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.229748725891113}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32308, 65520], "load": 1.0, "temperature": 52.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.54731731738212, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [10, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.048670768737793}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.42132994568185, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [11, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32308, 65520], "load": 0.24, "temperature": 51.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.186994552612305}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.42524746042346, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [12, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.304845809936523}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32308, 65520], "load": 1.0, "temperature": 51.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.578335047178435, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [13, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.2467522621154785}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32308, 65520], "load": 1.0, "temperature": 52.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.59417257261312, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [14, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.243383884429932}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32308, 65520], "load": 1.0, "temperature": 53.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.49724783682225, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [15, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.174020290374756}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.35153954637454, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [16, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32308, 65520], "load": 0.17, "temperature": 52.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.151361465454102}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.454943540418185, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [17, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.290616512298584}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32308, 65520], "load": 1.0, "temperature": 52.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.60629330191508, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [18, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.312309265136719}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32308, 65520], "load": 1.0, "temperature": 52.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.482230129465066, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [19, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.271075248718262}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32308, 65520], "load": 1.0, "temperature": 52.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.50502882401578, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [20, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.281228065490723}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.43485673696079, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [21, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32308, 65520], "load": 0.26, "temperature": 51.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.123138904571533}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.46947552476105, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [22, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.3131914138793945}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32308, 65520], "load": 1.0, "temperature": 52.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.470082707753654, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [23, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.26025390625}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32308, 65520], "load": 1.0, "temperature": 52.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.42389416567661, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [24, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.2235517501831055}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32308, 65520], "load": 1.0, "temperature": 52.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.516516110412105, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [25, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.311661720275879}, "pipe": "data"}
{"event": "line", "data": "Train: 0 [  31/32 (100%)]  Loss: 7.312 (7.24)  Time: 2.808s,   45.58/s  (2.912s,   43.95/s)  LR: 1.000e-05  Data: 0.001 (0.031)\n", "pipe": "stderr"}
{"event": "data", "data": {"rate": 45.50411972063367, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [26, 60]}, "pipe": "data"}
{"event": "line", "data": "Test: [   0/32]  Time: 1.126 (1.126)  Loss:  7.1176 (7.1176)  Acc@1:  0.0000 ( 0.0000)  Acc@5:  0.0000 ( 0.0000)\n", "pipe": "stderr"}
{"event": "line", "data": "Test: [  32/32]  Time: 0.375 (0.619)  Loss:  7.0510 (7.2337)  Acc@1:  0.0000 ( 0.0969)  Acc@5:  0.0000 ( 0.5329)\n", "pipe": "stderr"}
{"event": "line", "data": "Current checkpoints:\n", "pipe": "stderr"}
{"event": "line", "data": " ('/milabench/envs/extra/timm/redizilo.2023-05-08_17:54:51.224604/davit_large.D7/20230508-183305-davit_large-224/checkpoint-0.pth.tar', 0.09689922480620156)\n", "pipe": "stderr"}
{"event": "line", "data": "\n", "pipe": "stderr"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32308, 65520], "load": 0.64, "temperature": 52.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.583429980473035, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [27, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32559, 65520], "load": 0.03, "temperature": 53.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32559, 65520], "load": 0.22, "temperature": 53.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32559, 65520], "load": 0.04, "temperature": 54.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32559, 65520], "load": 0.13, "temperature": 53.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32559, 65520], "load": 0.06, "temperature": 53.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32559, 65520], "load": 0.17, "temperature": 53.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [3807, 65520], "load": 0.01, "temperature": 49.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.4138336181640625}, "pipe": "data"}
{"event": "line", "data": "Train: 1 [   0/32 (  0%)]  Loss: 7.414 (7.41)  Time: 3.388s,   37.78/s  (3.388s,   37.78/s)  LR: 1.008e-03  Data: 0.547 (0.547)\n", "pipe": "stderr"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32697, 65520], "load": 0.29, "temperature": 50.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.161294937133789}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.02465080812848, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [28, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.165196418762207}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32771, 65520], "load": 1.0, "temperature": 50.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.048356533627015, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [29, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.088315963745117}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32771, 65520], "load": 1.0, "temperature": 51.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.19939962682712, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [30, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.091288089752197}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32771, 65520], "load": 1.0, "temperature": 51.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.027502224961914, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [31, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.094071388244629}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32771, 65520], "load": 1.0, "temperature": 51.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.13111310808295, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [32, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.022368431091309}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.194433817799144, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [33, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32771, 65520], "load": 0.24, "temperature": 51.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.099251747131348}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.12821251650351, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [34, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.037964820861816}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32771, 65520], "load": 1.0, "temperature": 51.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.221716277342466, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [35, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.015068054199219}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32771, 65520], "load": 1.0, "temperature": 51.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.10725950574934, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [36, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.076755523681641}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32771, 65520], "load": 1.0, "temperature": 52.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.05481413566564, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [37, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.006778717041016}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32771, 65520], "load": 1.0, "temperature": 52.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.179953569324134, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [38, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.006064414978027}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.199290071983064, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [39, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32771, 65520], "load": 0.48, "temperature": 50.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.107608318328857}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.13267301402519, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [40, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.033906936645508}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32771, 65520], "load": 1.0, "temperature": 51.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.07787635253606, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [41, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.0689377784729}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32771, 65520], "load": 1.0, "temperature": 51.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.348816508605736, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [42, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 6.956124305725098}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32771, 65520], "load": 1.0, "temperature": 51.0}}}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.02729260393146, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [43, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 7.021805286407471}, "pipe": "data"}
{"event": "data", "data": {"rate": 45.3007862529747, "units": "items/s", "task": "train"}, "pipe": "data"}
{"event": "data", "data": {"task": "early_stop", "progress": [44, 60]}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"7": {"memory": [32771, 65520], "load": 0.5, "temperature": 51.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "train", "loss": 6.94273567199707}, "pipe": "data"}
{"event": "line", "data": "Memory access fault by GPU node-9 (Agent handle: 0x8833eb0) on address 0x7f9df4a00000. Reason: Unknown.\n", "pipe": "stderr"}
{"event": "end", "data": {"command": ["voir", "--config", "/milabench/envs/extra/timm/voirconf-davit_large.D7-0efae956f1553a76c1e03985181900f5.json", "/milabench/milabench/benchmarks/timm/pytorch-image-models/train.py", "--amp", "--model", "davit_large", "--batch-size", "128", "--lr-base", "0.01", "--data-dir", "/milabench/envs/data", "--dataset", "FakeImageNet", "--output", "/milabench/envs/extra/timm/redizilo.2023-05-08_17:54:51.224604/davit_large.D7", "--checkpoint-hist", "1"], "time": 1683570956.2752776, "return_code": -6}, "pipe": null}
