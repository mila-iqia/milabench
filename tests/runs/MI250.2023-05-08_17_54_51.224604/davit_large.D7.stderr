/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
Training with a single process on 1 device (cuda:0).
Model davit_large created, param count:196811752
Data processing configuration for current model + dataset:
	input_size: (3, 224, 224)
	interpolation: bicubic
	mean: (0.485, 0.456, 0.406)
	std: (0.229, 0.224, 0.225)
	crop_pct: 0.95
	crop_mode: center
Learning rate (0.005) calculated from base learning rate (0.01) and global batch size (128) with linear scaling.
Using native Torch AMP. Training in mixed precision.
Scheduled epochs: 300. LR stepped per epoch.
Train: 0 [   0/32 (  0%)]  Loss: 7.224 (7.22)  Time: 5.712s,   22.41/s  (5.712s,   22.41/s)  LR: 1.000e-05  Data: 0.755 (0.755)
Train: 0 [  31/32 (100%)]  Loss: 7.312 (7.24)  Time: 2.808s,   45.58/s  (2.912s,   43.95/s)  LR: 1.000e-05  Data: 0.001 (0.031)
Test: [   0/32]  Time: 1.126 (1.126)  Loss:  7.1176 (7.1176)  Acc@1:  0.0000 ( 0.0000)  Acc@5:  0.0000 ( 0.0000)
Test: [  32/32]  Time: 0.375 (0.619)  Loss:  7.0510 (7.2337)  Acc@1:  0.0000 ( 0.0969)  Acc@5:  0.0000 ( 0.5329)
Current checkpoints:
 ('/milabench/envs/extra/timm/redizilo.2023-05-08_17:54:51.224604/davit_large.D7/20230508-183305-davit_large-224/checkpoint-0.pth.tar', 0.09689922480620156)

Train: 1 [   0/32 (  0%)]  Loss: 7.414 (7.41)  Time: 3.388s,   37.78/s  (3.388s,   37.78/s)  LR: 1.008e-03  Data: 0.547 (0.547)
Memory access fault by GPU node-9 (Agent handle: 0x8833eb0) on address 0x7f9df4a00000. Reason: Unknown.
